modules/administration-guide/examples/snip_che-link-to-che-code-repository.adoc

```diff
@@ -0,0 +1,2 @@
+. Fork or download the Git link:https://github.com/che-incubator/che-code/tree/main/[repository] of Visual Studio Code - Open Source IDE for {prod}.
```

Comment:
I guess for the downstream docs the reference should be https://github.com/redhat-developer/devspaces-images/tree/devspaces-3-rhel-8/devspaces-code

Reply:
Checked. This is already in the downstream docs repo.